pubspec.yaml

```
@@ -88,4 +108,4 @@ flutter:
   #         weight: 700
   #
   # For details regarding fonts from package dependencies,
-  # see https://flutter.dev/to/font-from-package
+  # see https://flutter.dev/to/font-from-package
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)